index.js

```diff
@@ -280,6 +280,19 @@ function videoTrackSort(a, b) {
     return 1;
   }
 
+  if (a.format_id.includes('akfire_interconnect') || a.format_id.includes('fastly_skyfire')) {
+    // Vimeo video!
+    // If one has 'sep' in the format_id and one does not, we take the one with 'sep' in its format_id
+    // VIVI-12238: video tracks that don't have '_sep' in its format_id are sometimes failing, reasons unknown
+    if (a.format_id.includes('_sep') && !b.format_id.includes('_sep')) {
```

Comment:
Shouldn't we just filter them out entirely? Instead of sorting?

Reply:
@mattd233 you might be interested in this reply too - 

*sigh* vimeo videos don't follow any consistent pattern. We stumbled on this problem because customers reported some vimeo videos have stopped playing. Then Alysander went over our test collection and found that some (but not all) of the vimeo videos had stopped playing. 

So as an example, here's another vimeo video. This one is good... no broken tracks here. Also no tracks with "_sep" in the name. 

![image](https://github.com/viviedu/ytdl-process/assets/104745979/ca75cbe0-b832-4c1c-b33e-161681b90433)

Now look at BURRNESHA, which is one of the videos in our test collection that stopped working. 
![image](https://github.com/viviedu/ytdl-process/assets/104745979/402a1784-161e-42be-a80f-34bf5f196125)

So I'm taking the "least likely to cause new problems" approach here. My new code _only_ kicks in if a video happens to have two very similar tracks and one has "_sep" and one doesn't. In that case, we just prefer the one that has "_sep"

Reply:
Vime:omegalul:

Reply:
Seems good...